### PR TITLE
[Overflow-13] Sidebar Markup

### PR DESCRIPTION
**Pull request by IanUrriza10**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-13

## Commands
```
cd frontend
npm i
```
## Expected Output
Sidebar that has different looks for active, hover, and default.
Default is gray with no borders
Hover is red text
Active is black text with red border at right

## Screenshots
image

### CODE REVIEW

File: frontend/components/molecules/SidebarButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                   hover:text-[#ff2000] hover:border-0 hover:rounded-none
+                   active:text-black active:border-r-4 active:border-red-400 active:rounded-lg
+                   `}
+      >
+        {getIcon()}
+        <span className="pl-2">{Text}</span>
+      </Link>
+    </li>
+  );
+};
```
Comment (presyos) [inline frontend/components/molecules/SidebarButton/index.tsx:75]: Add an extra line before `export default SidebarButton;`

File: frontend/components/organisms/Sidebar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        Text="Tags"
+        isSelected={isSelected("/tags")}
+        url="/tags"
+      />
+      <SidebarButton
+        IconName="Teams"
+        Text="Teams"
+        isSelected={isSelected("/teams")}
+        url="/teams"
+      />
```
Comment (jescabillas) [inline frontend/components/organisms/Sidebar/index.tsx:48]: please refactor and use map in lodash to shorten the code

File: frontend/components/molecules/SidebarButton/index.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      return <HiCode />;
+    }
+    if (IconName === "Teams") {
+      return <HiUsers />;
+    }
+    if (IconName === "Users") {
+      return <HiUserGroup />;
+    }
+    return <></>;
+  };
```
Comment (jescabillas) [inline frontend/components/molecules/SidebarButton/index.tsx:54]: Can you please create atoms for this and refactor using switch case to shorten the code in this molecules

**Review by jescabillas — COMMENTED** [review]
Please check minor comments

**Review by presyos — APPROVED** [review]

**Pull request merged**